File: src/main/java/net/sharplab/epubtranslator/core/service/TranslateRequestChunk.java
package net.sharplab.epubtranslator.core.service;

import java.util.Collections;
import java.util.List;

/**
 * TranslateRequestChunk
 */
public class TranslateRequestChunk {

    public List<TranslateRequest> translateRequests;

    public TranslateRequestChunk(List<TranslateRequest> translateRequests) {
        this.translateRequests = Collections.unmodifiableList(translateRequests);
    }

    public List<TranslateRequest> getTranslateRequests() {
        return translateRequests;
    }
}

File: src/main/java/net/sharplab/epubtranslator/core/service/TranslateRequestChunk.kt
package net.sharplab.epubtranslator.core.service

import java.util.*

/**
 * TranslateRequestChunk
 */
class TranslateRequestChunk(translateRequests: List<TranslateRequest>) {
    val translateRequests: List<TranslateRequest> = Collections.unmodifiableList(translateRequests)
}
